src/faultlog/faultlog_policy.cpp

```diff
@@ -44,8 +44,8 @@ void FaultLogPolicy::populate(sdbusplus::bus::bus& bus, nlohmann::json& nagJson)
         }
         catch (const std::exception& ex)
         {
-            lg2::error("Failed to read allow_hw_isolation property {ERROR}",
-                       "ERROR", ex.what());
+            lg2::info("Failed to read allow_hw_isolation property {ERROR}",
+                      "ERROR", ex.what());
```

Comment:
ex.what is not needed, just ex is enough

Reply:
done